py/API_CHANGELOG.md

```diff
@@ -6,3 +6,16 @@ Note: that these changes only affect python-only code under `py/` path
 
 - `Serialization` not supported as canproto was removed. 
 
+- Changed all use of nupic to htm.   This means that Python users must import from
```

Comment:
ok, other option would be to have Python as a section in the other (only) API_CHangelog file

Reply:
+1 we should consolidate the C++ and Python changelogs into one big file with a section for each, and maybe a third section with general advice for porting code from the old nupic.